### PR TITLE
[1LP][RFR]  Automate dialog dynamic field refresh 

### PR DESCRIPTION
**Pull request by niyazRedhat**
### PRT Run
{{pytest: cfme/tests/services/test_dynamicdd_dialogelement.py::test_dialog_dynamic_field_refresh_in_log --long-running}}

### CODE REVIEW

File: cfme/tests/services/test_dynamicdd_dialogelement.py [hunk 1]
```diff
@@ -265,29 +266,45 @@ def test_dynamic_field_on_textbox(request, appliance, import_datastore, import_d
...
-            2. associate each to a different method that just logs "Refreshing X"
-            3. associating the dialog to a catalog item
-            4. tail -f log/automation.log | grep "Refreshing"
-            5. load the dialog
-            6. click "refresh" for field 2
-    Bugzilla:
-        1559999
+            1. Import Datastore and dialog
+            2. Add service catalog with above created dialog
+            3. Navigate to order page of service
```
Comment (ganeshhubale) [inline cfme/tests/services/test_dynamicdd_dialogelement.py]: First two steps seems to be of **setup** and 2nd steps should be like: ```Add service catalog with above imported dialog```

File: cfme/tests/services/test_dynamicdd_dialogelement.py [hunk 2]
```diff
@@ -265,29 +266,45 @@ def test_dynamic_field_on_textbox(request, appliance, import_datastore, import_d
...
+            4. In service Order page
+            5. Click on "refresh" for field 2
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5. Only text field2 should be refreshed in automation log
     """
```
Comment (ganeshhubale) [inline cfme/tests/services/test_dynamicdd_dialogelement.py]: Can you please confirm!
whether it is **field2** or **RefreshField2**

Comment (niyazRedhat) [reply cfme/tests/services/test_dynamicdd_dialogelement.py]: this is Field2

File: cfme/tests/services/test_dynamicdd_dialogelement.py (hunk 1 above)
Comment (niyazRedhat) [reply cfme/tests/services/test_dynamicdd_dialogelement.py]: Importing Datastore and dialog is testing steps

Comment (ganeshhubale) [reply cfme/tests/services/test_dynamicdd_dialogelement.py]: @niyazRedhat Can you please that step under **setup**?

Comment (niyazRedhat) [reply cfme/tests/services/test_dynamicdd_dialogelement.py]: added

**Pull request merged**